<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Invensile</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="informationTab">
       <attribute name="title">
        <string>Information</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QGroupBox" name="followUpBox">
          <property name="title">
           <string>Follow ups</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QTableView" name="followUpView"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QPushButton" name="pushButton">
                 <property name="text">
                  <string>Mark as correct</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_2">
                 <property name="text">
                  <string>Modify status</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Tags</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label">
                 <property name="toolTip">
                  <string/>
                 </property>
                 <property name="text">
                  <string>Filter tags</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="filterTagsEdit"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTableView" name="tagsView"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="newTagButton">
                 <property name="text">
                  <string>New tag</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="modifyTagButton">
                 <property name="text">
                  <string>Modify tag</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="deleteTagButton">
                 <property name="text">
                  <string>Delete tag</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Statuses</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="toolTip">
                  <string/>
                 </property>
                 <property name="text">
                  <string>Filter statuses</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="filterStatusesEdit"/>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QTableView" name="statusesView"/>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QPushButton" name="newStatusButton">
                 <property name="text">
                  <string>New status</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="modifyStatusButton">
                 <property name="text">
                  <string>Modify status</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="deleteStatusButton">
                 <property name="text">
                  <string>Delete status</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="searchTab">
       <attribute name="title">
        <string>Search</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="addressesTab">
       <attribute name="title">
        <string>Addresses</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Filter addresses</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="filterAddressEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="addressesView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="newAddressButton">
            <property name="text">
             <string>New address</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="viewAddressItemsButton">
            <property name="text">
             <string>View address items</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="modifyAddressButton">
            <property name="text">
             <string>Modify address</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteAddressButton">
            <property name="text">
             <string>Delete address</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="locationsTab">
       <attribute name="title">
        <string>Locations</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Filter locations</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="filterLocationsEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="locationsView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="newLocationButton">
            <property name="text">
             <string>New Location</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="viewItemsLocationButton">
            <property name="text">
             <string>View location items</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="modifyLocationButton">
            <property name="text">
             <string>Modify location</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteLocationButton">
            <property name="text">
             <string>Delete location</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="itemsTab">
       <attribute name="title">
        <string>Items</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Filter items</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="filterItemsEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="itemsView"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="newItemButton">
            <property name="text">
             <string>New item</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="modifyItemButton">
            <property name="text">
             <string>Modify item</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteItemButton">
            <property name="text">
             <string>Delete item</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionModify"/>
   </widget>
   <widget class="QMenu" name="menuCreate">
    <property name="title">
     <string>Create</string>
    </property>
    <addaction name="actionLocation"/>
    <addaction name="actionItem"/>
    <addaction name="actionAddress"/>
    <addaction name="separator"/>
    <addaction name="actionStatus"/>
    <addaction name="actionTag"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionRecent_Files"/>
    <addaction name="actionExport"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuCreate"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New file (Ctrl+N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open a file (Ctrl+O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save (Ctrl+S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>Close file (Ctrl+W)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionRecent_Files">
   <property name="text">
    <string>Recent Files</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit (Ctrl+Q)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy (Ctrl+C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste (Ctrl+V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionModify">
   <property name="text">
    <string>Modify</string>
   </property>
   <property name="toolTip">
    <string>Modify (Ctrl+M)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionLocation">
   <property name="text">
    <string>Location</string>
   </property>
   <property name="toolTip">
    <string>Create a location (Alt+C, L)</string>
   </property>
   <property name="shortcut">
    <string>Alt+C, L</string>
   </property>
  </action>
  <action name="actionItem">
   <property name="text">
    <string>Item</string>
   </property>
   <property name="toolTip">
    <string>Create an item (Alt+C, L)</string>
   </property>
   <property name="shortcut">
    <string>Alt+C, I</string>
   </property>
  </action>
  <action name="actionAddress">
   <property name="text">
    <string>Address</string>
   </property>
   <property name="toolTip">
    <string>Create an address (Alt+C, A)</string>
   </property>
   <property name="shortcut">
    <string>Alt+C, A</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionStatus">
   <property name="text">
    <string>Status</string>
   </property>
   <property name="shortcut">
    <string>Alt+C, S</string>
   </property>
  </action>
  <action name="actionTag">
   <property name="text">
    <string>Tag</string>
   </property>
   <property name="shortcut">
    <string>Alt+C, T</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
